ui/components/Layout.js

```diff
@@ -18,12 +18,11 @@ import Image from 'next/image'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import Script from 'next/script'
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
```

Comment:
@bananafo Why was `useEffect` added? Is it required by the new `wagmi` version?

Reply:
A bug was found during testing.The useAccount doesn't rerender on account change, so use useEffect to update account.View line 82.